project/MiMa.scala

```diff
@@ -35,7 +35,7 @@ object MiMa extends AutoPlugin {
           "10.0.4",
           "10.0.5"
       )
-        .map((version: String) => organization.value %% name.value % version)
+        .collect { case version if name.value != "akka-http-caching" => organization.value %% name.value % version }
```

Comment:
Let's put that module into a list of `ignoredModules` with a comment so it's clear what we are doing here.